src/emu/machine/vrc4373.h

```diff
@@ -115,12 +115,10 @@ class vrc4373_device : public pci_host_device {
 	UINT32 m_simm_base;
 	dynamic_array<UINT32> m_simm;
 
-
```

Comment:
Please restore this double blank line separating memory configuration from internal registers.